py/selenium/webdriver/common/options.py

```diff
@@ -127,7 +127,7 @@ def timeouts(self, timeouts: dict) -> None:
 
         :param timeouts: values in milliseconds for implicit wait, page load and script timeout
         """
-        if all(x in timeouts.keys() for x in ("implicit", "pageLoad", "script")):
+        if all(x in ("implicit", "pageLoad", "script") for x in timeouts.keys()):
```

Comment:
Thanks, I think this is a slightly better implementation than what existed before (Enforcing every timeout was provided else giving an ambiguous error) to now making sure all keys are in the set list, this is ok for now.  In future I think we should probably extract the 'known' keys and throwaway the rest without causing an error).